src/super_gradients/recipes/cityscapes_stdc_seg50.yaml

```diff
@@ -0,0 +1,107 @@
+#  STDC segmentation training example with Cityscapes dataset.
+#  Reproduction and refinement of paper: Rethinking BiSeNet For Real-time Semantic Segmentation.
+#
+#  Usage STDC1-Seg50:
+#      python -m torch.distributed.launch --nproc_per_node=4 train_from_recipe.py --config-name=cityscapes_stdc_seg50 external_checkpoint_path=<stdc1-backbone-pretrained-path>
+#  Usage STDC2-Seg50:
+#      python -m torch.distributed.launch --nproc_per_node=4 train_from_recipe.py --config-name=cityscapes_stdc_seg50 external_checkpoint_path=<stdc2-backbone-pretrained-path> architecture=stdc2_seg
+#
```

Comment:
Add comment about the resultion in inference

Reply:
Resolving this comment as we said this information should be in Readme (will add it with the other benchmark info)